docker/tds/docker-compose.yml

```diff
@@ -178,3 +180,14 @@ services:
     environment:
       - RABBITMQ_LOGS="-"
       - RABBITMQ_SASL_LOGS="-"
+
+  #Redis data store
+  redis:
```

Comment:
If we add this are we also going to change the local configurations to use redis rather than in memory?